src/styles/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__logo {
+    display: flex;
+  }
+}
+
+.nav {
+  &__list {
+    list-style: none;
+    display: flex;
+
+    &-item {
+      line-height: 60px;
+      position: relative;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $main-text-color;
+    font-size: 12px;
+    text-transform: uppercase;
+    cursor: pointer;
+    display: flex;
+    margin-left: 20px;
+    transition-duration: $animationTransitionDuration;
+
+    &--noMargin {
+      margin-left: 0;
+    }
```

Comment:
In this case, you have to specify in the html this class. You can set margins to "nav__list-item", and use the first or last child instead to get rid of the margin for the first element.